extensions/wikia/Helios/User.class.php

```diff
@@ -227,83 +240,105 @@ public static function onUserCheckPassword( $id, $username, $hash, $password, &$
 		return true;
 	}
 
-    /**
-     * Called in ExternalUser_Wikia registers a user.
-     *
-     * @param string $username string of the user name
-     * @param string $password string of the plaintext password the user entered
-     * @param string $email string of the user email
-     *
-     * @return boolean true on success, false otherwise
-     */
-    public static function register( $username, $password, $email, $birthdate )
-    {
-        $logger = \Wikia\Logger\WikiaLogger::instance();
-        $logger->info( 'HELIOS_REGISTRATION START', [ 'method' => __METHOD__ ] );
-
-        global $wgHeliosBaseUri, $wgHeliosClientId, $wgHeliosClientSecret;
-        $helios = new Client( $wgHeliosBaseUri, $wgHeliosClientId, $wgHeliosClientSecret );
+	/**
+	 * Called in ExternalUser_Wikia registers a user.
+	 *
+	 * @param string $username The username
+	 * @param string $password The plaintext password the user entered
+	 * @param string $email The user's email
+	 * @param string $birthDate
+	 *
+	 * @return bool true on success, false otherwise
+	 */
```

Comment:
There were a bunch of spaces that had to be converted to tabs in the next two functions.